solutions/fscloud/main.tf

```diff
@@ -0,0 +1,36 @@
+########################################################################################################################
+# Resource Group & Service ID
+########################################################################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.4"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+########################################################################################################################
+# AppID instance
+########################################################################################################################
+
```

Comment:
hmm, in order to be consistent with other DAs, we should probably add support to create a a KMS key in an existing KMS instance. For example see https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/30b001f78aac4b858d31a9e7dfec02db7be3220a/solutions/instances/main.tf#L25-L58

However if you want to create a GIT issue to add it as a follow up PR lets do that

Reply:
Created GIT issue: https://github.ibm.com/GoldenEye/issues/issues/8171